### PR TITLE
FT-02abcd: userCount, Route Active with purple BG, Add sub-header and text Employee, Add ember-qunit

### PR DESCRIPTION
**Pull request by HasanMusaraf**
a. Edit the users route create above to fetch all users and display number of records count in UI. Make `users` route as the default route.
b. Add users route link in the main left Nav and when the route is active, add the purple BG as shown in Design.
c. Add sub-header and add text `Employees`.
d. Add "ember-qunit": "^3.4.1" and remove ember-cli-qunit from package.json.

### CODE REVIEW

File: app/components/nav-listing.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { computed } from '@ember/object';
+
+export default Component.extend({
+  router: service(),
+
+  isUserRoute: computed('router.currentRouteName', function () {
+    return this.router.currentRouteName === 'users';
```
Comment (Shalini-Sivakumar) [inline app/components/nav-listing.js]: Use `get` here `get(this, 'router.currentRouteName')`

File: app/models/user.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import DS from 'ember-data';
+
+export default DS.Model.extend({
+  first_name: DS.attr('string'),
+  last_name: DS.attr('string'),
+  email: DS.attr('string'),
+  team: DS.attr('string'),
+  joiningDate: DS.attr('date')
```
Comment (Shalini-Sivakumar) [inline app/models/user.js]: Use camelCase in the model attribute names.

File: app/styles/app.scss [hunk 3]
```diff
@@ -10,9 +10,14 @@ body{
     background: #E5E5E5;
 }
 
+@mixin content-wrapper {
+  position: absolute;
```
Comment (Shalini-Sivakumar) [inline app/styles/app.scss:8]: why is position absolute required here?

File: app/styles/sub-styles/nav.scss [hunk 4]
```diff
@@ -18,6 +19,29 @@
 
 }
 
+.candidates {
+  position: absolute;
+  top: 66px;
+  left: 10px;
+  width: 40px;
+  height: 40px;
```
Comment (Shalini-Sivakumar) [inline app/styles/sub-styles/nav.scss:29]: why is position absolute required here? Already you used flex. Can you check?

File: app/components/nav-listing.js (hunk 1 above)
Comment (HasanMusaraf) [reply app/components/nav-listing.js]: I have fixed this change.

File: app/models/user.js (hunk 2 above)
Comment (HasanMusaraf) [reply app/models/user.js]: I have fixed this change.

**Review by Shalini-Sivakumar — APPROVED** [review]

**Pull request merged**